README.md

```diff
@@ -10,3 +10,126 @@
 [![license](https://img.shields.io/github/license/containers/oci-spec-rs.svg)](https://github.com/containers/oci-spec-rs/blob/master/LICENSE)
 
 ### Open Container Initiative (OCI) Specifications for Rust
+
+This library provides a convenient way to interact with the specifications defined by the [Open Container Initiative (OCI)](https://opencontainers.org). 
+
+- [Image Format Specification](https://github.com/opencontainers/image-spec/blob/main/spec.md)
+- [Runtime Specification](https://github.com/opencontainers/runtime-spec/blob/master/spec.md)
+- [Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/main/spec.md)
+
+```toml
+[dependencies]
+oci-spec = "0.5.0"
+```
+*Compiler support: requires rustc 1.54+*
+
+## Image Format Spec Examples
+- Load image manifest from filesystem
+```no_run
```

Comment:
Why don't you use `rust` instead of `no_run`?

Reply:
I left it out because then the the doc tests will also execute the code, which will fail because it tries to open files that do not exist. It seems you can do both though.

Reply:
cool!
https://github.com/containers/oci-spec-rs/blob/e518322b9ab7e9f0c2206ac8a77c06ce112de777/README.md